### ISSUE TITLE
Github Issues Check #3

### ISSUE THREAD

**Issue by caching13** [AguaClara/filter-constrictions#17]
Hi Filter Constrictions,
I did an Issues check for your team today, but since no significant Issues (thank you for adding labels for organizational purposes!) have been added since the previous check, your team will not be graded for this check. (I am giving the team the benefit of the doubt.) For clarification, other teams are getting grades for this check, but your team will not. Filter Constrictions will have a total of 4 checks before the end of the semester, whereas most other teams will have received 5 checks.

Please remember to update your Issues before the next check!

-Next Issues check will be in 2 weeks!
-Feel free to contact me with questions!

**Comment by JacquelineDokko** [comment]
@caseyching13 

Hi Casey, 

Thanks for checking in with us. However, we are left quite confused about the grading of the Issues section. 

We have been constantly updating our issues by commenting and closing completed tasks during nearly every meeting and experiment. Since we have not developed any new major issues, we have not created any new ones as the content of our project only necessitates (at least thus far) work on our already existing issues. I am not sure if you have access to our closed issues but please check them to see our recent progress. If that is not the problem then our question becomes: are we expected to be running into new problems constantly and thus creating new issues throughout the semester? How does it affect our grading if we have been successful in solving our initial issues rather than creating new ones?

Thank you, 
Filter Constrictions

**Comment by caching13** [comment]
Hi Filter Constrictions Team,

Sorry for any confusion! 

The last time I checked Issues for the different Subteams was right before Spring Break, and since I just did the check right after Spring Break, I am using this Issues check to check in on teams that needed improvements with their Issues. 

**Filter Constrictions is doing just fine and the team is doing what it is supposed to be doing!** For clarification, I did NOT give the team a grade of zero for this Issues check. Instead, I skipped this check for Filter Constrictions because it was not necessary. Issues check add up to 5% of your grade, so since I am doing 4 checks for Filter Constrictions, each check only counts as 1.25% of your grade (some teams will end up having 5 checks, so each check is 1% of their grade). 

It is not necessary to create new Issues if you are not running into problems. Hence, if the team continues to do what it has already been doing, do not worry!

Best,
Casey
